setup.py

```diff
@@ -86,7 +86,10 @@
         "Programming Language :: Python :: 3 :: Only",
     ],
     keywords="water systems, chemical engineering, process modeling, filtration, desalination, nawi",
-    packages=find_namespace_packages(),
+    # just include watertap and everything under it
+    packages=find_namespace_packages(
+        include=("watertap*",),
+    ),
```

Comment:
To clarify:

- I think the point being made in #879 is valid, and we should modify the `setup.py` settings so that files and directories other than `watertap` don't end up in the installation
- However, since this change would impact other things beyond the original scope of this PR (and I'm planning to rework our distribution setup soon), it might make sense to deal with it separately
- That said, ultimately I don't have strong preferences and I'm reasonably confident that our tests would catch any possible regression introduced by this change, so I'd be OK if we wanted to merge it as part of this PR

Reply:
Although it may look a bit brittle, all of the tests are passing now. As for this change, I'll let @bknueven comment on if there's a strong reason to keep it.

Reply:
Yeah, I agree, I don't think there's any harm in merging this as-is (including the change in `setup.py`).